mujoco_py/mjviewer.py

```diff
@@ -280,6 +280,8 @@ def _create_full_overlay(self):
                          (1 / self._time_per_render, extra))
         self.add_overlay(const.GRID_BOTTOMLEFT, "Solver iterations", str(
             self.sim.data.solver_iter + 1))
+        step = round(self.sim.data.time / self.sim.model.opt.timestep)
+        self.add_overlay(const.GRID_BOTTOMRIGHT, "step", str(step))
```

Comment:
Nit: We capitalize all other labels, so we should do it here too. I.e. "step" -> "Step"